src/pysdmx/model/__base.py

```diff
@@ -0,0 +1,212 @@
+from datetime import datetime
+from typing import Any, Dict, Optional, Sequence, Union
+
+from msgspec import Struct
+
+
+class Annotation(Struct, frozen=True, omit_defaults=True):
```

Comment:
We could add a post init check, making sure that at least one of the properties has been populated? Else, it would be possible to create totally empty annotations. Just a minor comment so, feel free to ignore 😊

Reply:
@sosna please add this after the PR.

Reply:
Mentioned in #32